source/reference/hyperlinks.txt

```diff
@@ -0,0 +1,81 @@
+.. _style-guide-hyperlinks:
+
+==========
+Hyperlinks
+==========
+
+You can link out to other sites, internal pages or sections using inline
+hyperlinks and cross-referenced hyperlinks.
+
+Inline Hyperlinks
+-----------------
+
+Use inline hyperlinks when you want to link to a full URL.
+
+Example
+~~~~~~~
+
+.. code-block:: rst
+
+   Here is the `MongoDB Go Driver Documentation <https://www.mongodb.com/docs/drivers/go/current/>`__.
+
+Rendered View
+`````````````
+
+Here is the `MongoDB Go Driver Documentation <https://www.mongodb.com/docs/drivers/go/current/>`__.
+
+Cross-Referenced Hyperlinks
+---------------------------
+
+Use cross-referenced hyperlinks when you want to link to another page in
+docs. This page can be in the same repo, or another within the same
+organization.
+
+Procedure
+~~~~~~~~~
+
+.. procedure::
+   :style:  normal
+
+   .. step:: If the hyperlink doesn't already exist, in the line before where you want to link to, create the cross-referenced hyperlink.
+
+      If the hyperlink is in a different repo, add the rendered docs URL
+      landing page and append ``objects.inv``. Add the URL to the
+      ``intersphinx`` array in the ``snooty.toml`` file.
```

Comment:
As a newcomer to hyperlinks, I'm not sure what it means for a hyperlink to (not) exist. Does that mean it hasn't been added to snooty.toml yet? And I'm confused by "in the line before where you want to link to." :)

Is the procedure different if the hyperlink is in the same repo?

Reply:
So sometimes, someone might have already put a hyperlink there, which means you just need to reference it.  But I'll remove the part about a line before since its sort of implied.